### PR TITLE
Add methods to get and set time, temperature unit and comfort level

### PR DESCRIPTION
**Pull request by qwandor**
^_^

Work towards #49.

### CODE REVIEW

File: mijia/src/bluetooth.rs [hunk 1]
```diff
@@ -226,6 +226,43 @@ impl BluetoothSession {
             .await
             .wrap_err_with(|| format!("disconnecting from {:?}", id))
     }
+
+    /// Read the value of the characteristic of the given device with the given path. The path
+    /// should be of the form "/service0001/char0002".
+    pub async fn read_characteristic_value(
+        &self,
+        id: &DeviceId,
+        characteristic_path: &str,
```
[the remark this answers is not in the document]
Comment (qwandor) [reply mijia/src/bluetooth.rs:235]: Made `pub(crate)` for now. I agree that looking up by UUID is the right way to do this, maybe cached somehow, but I couldn't be bothered for now.

File: mijia/examples/properties.rs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                "Time: {}, Unit: {}, Comfort level: {}",
+                sensor_time, temperature_unit, comfort_level
+            );
+        }
+    }
+
+    Ok(())
+}
+
+fn include_sensor(sensor: &SensorProps, filters: &Vec<String>) -> bool {
```
[the remark this answers is not in the document]
Comment (qwandor) [reply mijia/examples/properties.rs]: Done.

File: mijia/examples/properties.rs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+const SCAN_DURATION: Duration = Duration::from_secs(5);
+
+#[tokio::main]
+async fn main() -> Result<(), eyre::Error> {
+    pretty_env_logger::init();
+
+    // If at least one command-line argument is given, we will only try to connect to sensors whose
+    // MAC address containts one of them as a sub-string.
+    let filters: Vec<_> = env::args().collect();
```
[the remark this answers is not in the document]
Comment (qwandor) [reply mijia/examples/properties.rs]: Done.

File: mijia/src/decode.rs [hunk 4]
```diff
@@ -80,4 +210,65 @@ mod tests {
...
+    fn decode_time_too_long() {
+        assert!(decode_time(&[0x01, 0x02, 0x03, 0x04, 0x05]).is_err());
+    }
+
+    #[test]
+    fn encode_decode_time() {
+        let time = SystemTime::UNIX_EPOCH + Duration::from_secs(12345678);
+        assert_eq!(decode_time(&encode_time(time).unwrap()).unwrap(), time);
+    }
 }
```
[the remark this answers is not in the document]
Comment (qwandor) [reply mijia/src/decode.rs]: Done.

File: mijia/examples/properties.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (qwandor) [reply mijia/examples/properties.rs]: Done the checking bit. I find the substring matching handy as I can just type the last few digits.

**Review by alsuren — APPROVED** [review]
🐳

**Pull request merged**